docs/user/gcp/quota.md

```diff
@@ -0,0 +1,13 @@
+# Handling quota on GCP
+
+All cloud providers have some concept of limits imposed to protect their services from misuse as well as protect customers from mistakes that can leads to excessive charges.  In GCP these will vary by region, project and account.  This is great news in terms of flexibility but can present an initial hurdle if the defaults in your project are too low.
+
+## Common problems
+The most likely areas of quota contention are for CPU, Static IPs and Storage (SSD).  A vanilla IPI installation will result in 24 CPUs, 3 Static IPs and 768 GB of storage consumed.  To be clear, that's the absolute bare minimum.  It's critical to leave enough headroom for growth especially for projects shared by multiple teams.  Whenever an installation failes it's best to check `.openshift_install.log` for errors.
```

Comment:
> `To be clear, that's the absolute bare minimum.`

This would be un-necessary if we regrouped according to https://github.com/openshift/installer/pull/2315/files#r320993968